Content.Shared/Doors/Systems/PryingSystem.cs

```diff
@@ -0,0 +1,131 @@
+using Content.Shared.Doors.Prying.Components;
+using Content.Shared.DoAfter;
+using Robust.Shared.Serialization;
+using Content.Shared.Administration.Logs;
+using Content.Shared.Database;
+using Content.Shared.Doors.Components;
+using Content.Shared.Doors.Systems;
+using Content.Shared.Tools.Components;
+
+namespace Content.Shared.Doors.Prying.Systems;
+public class DoorPryingSystem : EntitySystem
+{
```

Comment:
Please make this sealed as per the warning and add a space with the namespace.

This should also be called PryingSystem not DoorPryingSystem as it shouldn't be hardcoded to doors.